lib/pulse/lib/Pulse.Lib.FlippableInv.fsti

```diff
@@ -27,5 +27,20 @@ val mk_finv (p:slprop { is_storable p }) : stt (finv p) emp (fun x -> off x)
 
 val iname_of #p (f : finv p) : erased iname
```

Comment:
iname is erasable now (it wasn't previously). So, we could just remove the erased annotation on the result type here

Reply:
Ah indeed, thanks! That fixes it. I was thinking the coercion should kick in anyway in the Pulse code below, but that's not the case since we check them at type `inames`, not `list iname`, so `iname_list` does not trigger.